x-pack/platform/plugins/shared/fleet/public/applications/integrations/sections/epm/components/package_list_grid/grid.tsx

```diff
@@ -120,7 +120,9 @@ export const GridColumn = ({
           }
         }}
         scrollElement={
-          scrollElementId ? document.getElementById(scrollElementId) ?? undefined : undefined
+          (scrollElementId && document.getElementById(scrollElementId)) ||
+          document.getElementById('app-main-scroll') ||
```

Comment:
Should we make these new Ids constants, so that we can can define their values once and not have to run a search and replace should it change, also applies to previous PR (https://github.com/elastic/kibana/pull/226568/files#r2185043771)?